ios/browser/api/bookmarks/BUILD.gn

```diff
@@ -19,10 +19,12 @@ source_set("bookmarks") {
   deps = [
     "//base",
     "//components/bookmarks/browser",
+    "//components/prefs:prefs",
```

Comment:
`:prefs` is redundant. Same with many others in this change.

Reply:
Pretty sure I added this after warnings were raised in gn check